microsoft-excel-online/1.0/modules/ROOT/pages/microsoft-excel-online-connector-reference.adoc

```diff
@@ -3846,26 +3996,28 @@ Microsoft Excel Online Connector API has some limitations. See the following lin
 * EXCEL:SERVER_ERROR
 * EXCEL:TIMEOUT
 
+[[RangeGetUsedRange]]
+== Range Get Used Range
+`<excel:range-get-used-range>`
 
-[[RangeMerge]]
-== Range Merge
-`<excel:range-merge>`
+
+Returns the used range of the given range object.
 
 
 === Parameters
 
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | | x
-| Workbook Id a| String |  The ID of the workbook. |  | x
-| Name a| String |  The name. |  |
+| Configuration | String | The name of the configuration to use. | | *x*
+| Workbook Id a| String |  The workbook ID of where the worksheet is located. |  | *x*
```

Comment:
Maybe:
The ID of the workbook where the worksheet is located.

Reply:
Got it! This specific description only appears once in the reference so an easy fix.